samples/react-chat/src/App.tsx

```diff
@@ -26,15 +26,7 @@ const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChat) => vo
     }
   ];
 
-  return (
-    <Get
-      resource="me/chats?$expand=members,lastMessagePreview&orderby=lastMessagePreview/createdDateTime desc"
-      scopes={['chat.read']}
-      cacheEnabled={false}
-    >
-      <ChatList menuItems={menus} buttonItems={buttons} onSelected={onSelected} />
-    </Get>
-  );
+  return <ChatList chatThreadsPerPage={3} menuItems={menus} buttonItems={buttons} onSelected={onSelected} />;
```

Comment:
Shouldn't "chatThreadsPerPage" be pulled from that variable.

Reply:
um, which variable?

Reply:
Nevermind, I saw chatThreadsPerPage in ChatList, but I didn't notice that was the prop for here. this is fine.